frontend/src/metabase/static-viz/components/Gauge/utils.unit.spec.ts

```diff
@@ -0,0 +1,177 @@
+import type { NumberFormatOptions } from "metabase/static-viz/lib/numbers";
```

Comment:
@alxnddr unit tests added.

Certain functions aren't tests
- `calculateValueFontSize`: Rely on `measureText` which doesn't make much sense in unittest.
- `calculateChartScale`: Rely on `measureText` which doesn't make much sense in unittest. 
- `gaugeAccessor`: Trivial
-  `gaugeSorter`: Trivial
-  `colorGetter`: Trivial

Maybe the last 3 functions should have been inline instead 🤔

Reply:
Awesome, thanks!